### DISCUSSION TITLE
custom prefix not working as expected with non semantic

### DISCUSSION THREAD

**Discussion by wyardley** [renovatebot/renovate#16275] [Request Help]
I use a custom commit prefix with the following `commitMessagePrefix` (https://docs.renovatebot.com/configuration-options/#commitmessageprefix) in the config:
```
"commitMessagePrefix": "[deps]",
```
which would previously result in a commit message like this:

```
[deps] update Terraform foo
```
Recently, it shifted to look like this instead:

```
[deps]: [deps] Update Terraform foo
```

Wondering if it could be related to https://github.com/renovatebot/renovate/pull/16239? I can try explicitly disabling `semanticCommits`, but verified that it's not set in our org preset that's inherited.

**Comment by wyardley** [comment]
I see - dupe of https://github.com/renovatebot/renovate/issues/16269

**Comment by rarkins** [comment]
Reverted, sorry. Will be released soon
